lessons/introduction-to-smartpy/13/13.mdx

```diff
@@ -156,7 +156,7 @@ Let's start:
 2. In the `sp.verify`'s condition field: 
     * use `self.data.bot_manager` to access the address stored inside `bot_manager`. 
     * check it's equivalency with `sp.sender`:
-       * SmartPy automatically populates `sp.address` with the address that called the current entry point.
+       * SmartPy automatically populates `sp.sender` with the address that called the current entry point.
```

Comment:
Hi @STFinancial ,
> I'm not sure this is correct, but I was confused by this line so I assume it's supposed to be sp.sender

The address that calls the current entry point is referred to as sp.sender inside the smart contract so yes, it should be **sp.sender**.   

Nice catch there 🎉